### ISSUE TITLE
v-for issue with async content

### ISSUE THREAD

**Issue by ux-leap** [alvarotrigo/vue-fullpage.js#131]
Hi,
How do I use vue-fullpage.js with v-for? If I do this way the content is generated but fullpage.js throws an error:

> Error in mounted hook: "TypeError: Cannot read property 'parentNode' of undefined"

**Some Page Component**
```
<template>
<full-page :options="optionsFullPage" id="fullpage">
    <section v-for="(section, index) in sections" :key="index" class="section">
        {{sectionTitle}}
    </section>
</full-page>
</template>

export default {
    name: "Page",
    data() {
        return {
            sections: [],
            optionsFullPage: {
                navigationTooltips: [],
                easingcss3: "ease-in-out",
                afterLoad: (origin, destination, direction) => {},
                onLeave: (origin, destination, direction) => {}
            }
        };
    },
    methods: {
        getContent() {
            // Async content
            this.$prismic.client.getByUID("section", "section").then(document => {
                this.sections = document.data.body;
                // this.$refs.fullpage.build();
            });
        }
    },
    created() {
        this.getContent();
    }
};
```

Thanks

**Comment by alvarotrigo** [comment]
> If I do this way the content is generated but fullpage.js throws an error:

Can you please provide a reproduction of the issue in codesanbox or similar?
Also, can you paste the complete error info with line and file too?

**Comment by ux-leap** [comment]
Hi @alvarotrigo 

Here you go: [Codesanbox Test](https://codesandbox.io/s/vue-fullpagejs-issue-109-vtgyr)

Thanks

**Comment by alvarotrigo** [comment]
Please see your codepen updated:
https://codesandbox.io/s/vue-fullpagejs-issue-109-4m0ej

Note:

- I added a section when there are no sections yet, as it is compulsory to have at least one section.

```js
      <section v-if="slices.length==0" class="section">Test</section>

```

- When you display your sections, one should contain the class `active` so I added it:

```js
v-bind:class="{ active: !index }"
```

- To react after the DOM has been updated and call the `build` function I used the `updated` callback:

```js
  updated: function() {
    // Code that will run only after the
    // entire view has been re-rendered
    this.$refs.fullpage.build();
  },
``` 

@vaso2, @dobromir-hristov do you think we can perhaps build these improvements directly into the vue-fullpage.js component so people won't have to manually deal with them?

**Comment by ux-leap** [comment]
@alvarotrigo thanks for your help and explanation. Everything works fine now!

**Comment by dobromir-hristov** [comment]
The #140 PR should help with this.

**Comment by alvarotrigo** [comment]
Fixed in 0.1.5!
